users/management/commands/promote_user.py

```diff
@@ -0,0 +1,85 @@
+"""Promote/demote a user."""
+
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """Promote/demote a user."""
+
+    help = "Promote/demote a user."
+
+    def add_arguments(self, parser):
+        """Add arguments to the command."""
+
+        parser.add_argument(
+            "email",
+            type=str,
+            help="The email of the user to promote/demote.",
+        )
+
+        parser.add_argument(
+            "--promote",
+            action="store_true",
+            help="Promote the user to staff.",
+        )
+
+        parser.add_argument(
+            "--demote",
+            action="store_true",
+            help="Demote the user from staff.",
+        )
+
+        parser.add_argument(
+            "--superuser",
+            action="store_true",
+            help="Promote the user to superuser.",
+        )
```

Comment:
I would suggest structuring this using subparsers because it'll handle some of the validation you're doing below automatically. I recently did this in open-discussions for a similar type of command and it simplifies things substantially: https://github.com/mitodl/open-discussions/blob/master/authentication/management/commands/validate_user_emails.py#L30